kik_unofficial/client.py

```diff
@@ -33,7 +32,7 @@ class KikClient:
     """
 
     def __init__(self, callback: callbacks.KikClientCallback, kik_username, kik_password,
-                 kik_node=None, device_id=random_device_id(), android_id=random_android_id()):
+                 kik_node=None, device_id=random_device_id(), android_id=random_android_id(), logger=False):
```

Comment:
Let's call this `logging` instead of `logger`

Reply:
I did that at first, but there is a module called logger, so i thought logging would be better, ill switch it back tho